labman/db/support_files/populate_test_db.sql

```diff
@@ -935,48 +1049,51 @@ BEGIN
             INSERT INTO labman.container (container_type_id, latest_upstream_process_id, remaining_volume)
-                VALUES (well_container_type_id, plating_process_id, 10)
+                VALUES (well_container_type_id, curr_plating_process_id, 10)
                 RETURNING container_id INTO plating_container_id;
             INSERT INTO labman.well (container_id, plate_id, row_num, col_num)
-                VALUES (plating_container_id, sample_plate_id, idx_row_well, idx_col_well);
+                VALUES (plating_container_id, curr_sample_plate_id, idx_row_well, idx_col_well);
             INSERT INTO labman.composition (composition_type_id, upstream_process_id, container_id, total_volume)
-                VALUES (sample_comp_type_id, plating_process_id, plating_container_id, 10)
+                VALUES (sample_comp_type_id, curr_plating_process_id, plating_container_id, 10)
                 RETURNING composition_id INTO plating_composition_id;
             INSERT INTO labman.sample_composition (composition_id, sample_composition_type_id, sample_id, content)
                 VALUES (plating_composition_id, plating_sample_comp_type_id, plating_sample_id, plating_sample_content)
                 RETURNING sample_composition_id INTO plating_sample_composition_id;
 
             CONTINUE WHEN idx_row_well = 8 AND idx_col_well = 12;
 
-            -- GDNA WELLS
+            -- container, well, and composition for the current sample on gdna plate
+            -- (note there are 4 of these plates--hence use of curr_gdna_plate_id)
             INSERT INTO labman.container (container_type_id, latest_upstream_process_id, remaining_volume)
-                VALUES (well_container_type_id, gdna_process_id, 10)
+                VALUES (well_container_type_id, curr_gdna_process_id, 10)
                 RETURNING container_id INTO gdna_container_id;
             INSERT INTO labman.well (container_id, plate_id, row_num, col_num)
-                VALUES (gdna_container_id, gdna_plate_id, idx_row_well, idx_col_well);
+                VALUES (gdna_container_id, curr_gdna_plate_id, idx_row_well, idx_col_well);
             INSERT INTO labman.composition (composition_type_id, upstream_process_id, container_id, total_volume)
-                VALUES (gdna_comp_type_id, gdna_process_id, gdna_container_id, 10)
+                VALUES (gdna_comp_type_id, curr_gdna_process_id, gdna_container_id, 10)
                 RETURNING composition_id INTO gdna_comp_id;
             INSERT INTO labman.gdna_composition (composition_id, sample_composition_id)
                 VALUES (gdna_comp_id, plating_sample_composition_id)
                 RETURNING gdna_composition_id INTO gdna_subcomposition_id;
 
-            -- 16S LIBRARY PREP WELLS
+            -- primer for the current sample's position on the EMP 16S primer plate 1
+            -- TODO: Should I change this to use different primer plates for the four gdna plates?
```

Comment:
Not sure

Reply:
Not sure either but I guess this also can be a new issue ... if we need it.